src/components/RefinementList.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <div :class="suit('')" v-if="state">
```

Comment:
The class `.ais-RefinementList--noRefinement` is missing when the component cannot be refined.